<?xml version="1.0"?>
<launch>

	<!-- Launch arguments -->
	<arg name="dice_params" default="$(find drims_template)/config/params.yaml"/>
	
	<!-- Loading config file to param server -->
	<rosparam command="load" file="$(arg dice_params)"/>

	<!-- Load main node -->
  	<node name="test_example_node" pkg="drims_template" type="test_example" output="screen">	
	</node>
</launch>